### PR TITLE
Feature/reddit home page

### PR DESCRIPTION
**Pull request by Pawelooo**

### CODE REVIEW

**Comment by Pawelooo** [comment]
This is what the website looks like
image

File: templates/base.html [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+                {% endfor %}
+            </ul>
+        {% endif %}
+
+        {% block content %}
+        {% endblock %}
+
+        <footer class="mt-auto bg-dark text-light text-center py-3 navbar-top">
+            <p>&copy; 2024 PyMasters.pl. All rights reserved.</p>
+{#            <p><a href="{% url 'privacy-policy' %}" class="text-decoration-none link-light">Privacy policy</a></p>#}
```
Comment (jacoor) [inline templates/base.html]: Placeholder for privacy policy is part of this task - so this is not correct.

**Review by jacoor — CHANGES_REQUESTED** [review]
This is not working
image
Please make it home page - add required template view.

**Comment by Pawelooo** [comment]
image
Okay, now it's on the main page

File: articles/views.py [hunk 2]
```diff
@@ -0,0 +1,6 @@
+from django.http import HttpRequest
+from django.shortcuts import render
+
+
+def home_view(request: HttpRequest) -> HttpRequest:
```
Comment (jacoor) [inline articles/views.py]: Via readme: prefer class based views.

**Review by jacoor — CHANGES_REQUESTED** [review]

**Review by jacoor — APPROVED** [review]

**Pull request merged**